### PR TITLE
examples/kubernetes-ingress/scripts: factor out cri-o installation into function

### PR DESCRIPTION
**Pull request by ianvernon**
Since cri-o is now installed in the base image used in the Vagrantfile, do not
try to install it when provisioning the K8s VMs. Factor out this logic into
a separate function instead of removing it in case it is needed in the future.

Signed-off by: Ian Vernon <ian@cilium.io>

Fixes: #4362 

**How to test (optional)**:
e.g. Run `K8S=1 ./contrib/vagrant/start.sh` . The VM should boot without error.

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/4382)

### CODE REVIEW

**Comment by ianvernon** [comment]
test-me-please

File: examples/kubernetes-ingress/scripts/03-install-kubernetes-worker.sh [hunk 1]
```diff
@@ -11,12 +11,28 @@ dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
...
 
-certs_dir="${dir}/certs"
+   cat <<EOF > /etc/apt/sources.list.d/projectatomic-ubuntu-ppa-artful.list
+deb http://ppa.launchpad.net/projectatomic/ppa/ubuntu xenial main
+deb-src http://ppa.launchpad.net/projectatomic/ppa/ubuntu artful main
+EOF
+   sudo apt-get update
+   sudo apt-get install cri-o-1.10 -y
+
+   download_to "${cache_dir}/containerd" "cri-containerd-1.1.0.linux-amd64.tar.gz" \
```
Comment (aanm) [inline examples/kubernetes-ingress/scripts/03-install-kubernetes-worker.sh]: split this in 2 functions. From line 27 to line 32 we are installing `containerd`. `crio` is installed from line 18 to line 25

**Review by aanm — DISMISSED** [review]

File: examples/kubernetes-ingress/scripts/03-install-kubernetes-worker.sh (hunk 1 above)
Comment (ianvernon) [reply examples/kubernetes-ingress/scripts/03-install-kubernetes-worker.sh]: Done.

**Review by aanm — APPROVED** [review]
tky

**Comment by aanm** [comment]
test-me-please

**Pull request merged**